components/observations/SimpleForm.tsx

```diff
@@ -67,10 +45,12 @@ const useKeyboardVerticalOffset = () => {
   return useHeaderHeight();
 };
 
-export const SimpleForm: React.FC<{
+interface SimpleFormProps {
   center_id: AvalancheCenterID;
   onClose?: () => void;
-}> = ({center_id, onClose}) => {
+}
+
+export const SimpleForm: React.FC<SimpleFormProps> = ({center_id, onClose}: SimpleFormProps) => {
```

Comment:
This same lint false negative has happened a couple times, not sure what's going on.

```
/avy/components/observations/SimpleForm.tsx
  53:56  error  'center_id' is missing in props validation  react/prop-types
  53:67  error  'onClose' is missing in props validation    react/prop-types

✖ 2 problems (2 errors, 0 warnings)
```